code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -892,12 +893,11 @@
 	if(refund) charge_time *= 2
 	return TRUE
 
-
 /obj/item/weapon/gun/energy/yautja/plasma_caster
 	name = "plasma caster"
 	desc = "A powerful, shoulder-mounted energy weapon."
-	icon_state = "plasma"
-	item_state = "plasma_wear"
+	var/dummy_icon_state = "plasma"
+	var/dummy_item_state = "plasma_wear"
```

Comment:
call this base_xxxx_state instead, dummy is a bad name for what it's used for

Reply:
No problem; thank you.